lib/args.js

```diff
@@ -32,13 +32,16 @@ module.exports = function(options) {
     for (var key in options) {
       var opt = options[key];
       if (typeof opt === 'object') {
+        yargs
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.